openstates/ga/legislators.py

```diff
@@ -50,6 +50,20 @@ def scrape_session(self, term, chambers, session):
         for member in members:
             guid = member['Id']
             member_info = backoff(self.sservice.GetMember, guid)
+
+            # Check to see if the member has vacated; skip if so:
+            try:
```

Comment:
Hey Richard,

Generally, I'm not opposed to making things more functional, but for the Open States code base, I'd like to avoid these sorts of function compositions with lambdas if possible for a number of reasons.  I would consider something like the following more fitting:
```
legislative_service = next(service for service in
    member_info['SessionsInService']['LegislativeService’]
    if service['Session']['Id'] == sid)
```
Otherwise, everything checks out.
Everything checks out, though.